examples/relayer-load-balance/index.js

```diff
@@ -0,0 +1,64 @@
+require('dotenv').config();
+
+const { Defender } = require('@openzeppelin/defender-sdk');
+
+async function loadbalance() {
+  const LOAD_BALANCE_THRESHOLD = 50;
+  const relayerCredsForMainNet = [
+    {
+      relayerApiKey: process.env.RELAYER_API_KEY_1,
+      relayerApiSecret: process.env.RELAYER_API_SECRET_1,
+    },
+    {
+      relayerApiKey: process.env.RELAYER_API_KEY_2,
+      relayerApiSecret: process.env.RELAYER_API_SECRET_2,
+    },
+  ];
+  const relayerClientsForMainNet = relayerCredsForMainNet.map((creds) => new Defender(creds));
+
+  const getNextAvailableRelayer = async () => {
+    for (const client of relayerClientsForMainNet) {
+      const relayerStatus = await client.relaySigner.getRelayerStatus();
+      if (relayerStatus.numberOfPendingTransactions < LOAD_BALANCE_THRESHOLD) {
```

Comment:
The threshold we defined in docs is 50 txs/min, I would compare the `nonce (1min ago) - nonce now` and if  > 50 then switch relayer. But maybe that adds some complexity to the example 🤔